src/Examples/Advanced/CustomUnitModels/methanol_param_VLE.py

```diff
@@ -127,35 +122,37 @@ def build(self):
 
         self.gamma = Param(within=NonNegativeReals, mutable=True, default=gamma, doc='Ratio of Cp to Cv')
 
-        self.Cp = Param(within=NonNegativeReals, mutable=True, default=Cp, doc='Constant pressure heat capacity [MJ/(kgmol K)]')
+        self.Cp = Param(within=NonNegativeReals, mutable=True, default=Cp,
+#                        units=pyunits.MJ/pyunits.kmol/pyunits.K,
+                        doc='Constant pressure heat capacity [MJ/(kmol K)]')
 
     @classmethod
     def define_metadata(cls, obj):
         """Define properties supported and units."""
         obj.add_properties(
-            {'flow_mol': {'method': None, 'units': 'kgmol/s'},
+            {'flow_mol': {'method': None, 'units': 'kmol/s'},
```

Comment:
The "units" section of the metadata is not really needed anymore, as units are fully defined by the units metadata below. You could just remove this from all entries.